src/amoc_api.erl

```diff
@@ -0,0 +1,27 @@
+-module(amoc_api).
```

Comment:
We'll want to refine this API befe merging. 
* we have too many handlers
* we are dispatching requests based on the handlers state which is bad; we want to do that based on the matches like /scenario/:action read [here](http://ninenines.eu/docs/en/cowboy/2.0/guide/routing/)
* we want to better separte of concerns: for example one API for dealing with scenarios solely and the other one for the amoc cluster (ping node, healtcheck etc)

I think that we'll came up with the plan after meeting on the HTTP APIs with our colleague.